### DISCUSSION TITLE
Preventing SQL Injection

### DISCUSSION THREAD

**Discussion by AlexiJemano** [tursodatabase/libsql#1780] [Q&A]
I was wandering if turso node.js module has any build-in solution for SQL injection, if not what is the best solution for the problem?

**Comment by avinassh** [comment]
Hey @AlexiJemano, you can use parameterised statements to prevent SQL injection. Like:

```
await turso.execute({
  sql: "SELECT * FROM users WHERE id = ?",
  args: [1],
});
``` 

from: https://docs.turso.tech/sdk/ts/quickstart
